File: 1.study/6.input_output/1-1.py
# -*- coding: utf-8 -*-
in_str = input("아이디를 입력해주세요.\n")
# print(type(in_str))
real_kst = "11"
real_gst = "ab"
if real_kst == in_str:
    print("Hello, kst")
elif real_gst == in_str:
    print("Hello, gst")
else:
    print("Who are you?")

File: 1.study/6.input_output/1-1.rb
puts("아이디를 입력해주세요")
input = gets.chomp()
real_kst = "11"
real_gst = "ab"
if real_kst == input
    puts("Hello, kst")
elsif real_gst == input
    puts("Hello, gst")
else
    puts("Who are you?")
end